test/integration/examples/apiserver_test.go

```diff
@@ -226,30 +230,40 @@ func TestAPIServiceWaitOnStart(t *testing.T) {
 }
 
 func TestAggregatedAPIServer(t *testing.T) {
+	// Testing default, BanFlunder default=true in 1.2
 	t.Run("WithoutWardleFeatureGateAtV1.2", func(t *testing.T) {
-		testAggregatedAPIServer(t, false, "1.2")
+		testAggregatedAPIServer(t, false, true, "1.2", "1.2")
 	})
+	// Testing emulation version N, BanFlunder default=true in 1.1
 	t.Run("WithoutWardleFeatureGateAtV1.1", func(t *testing.T) {
-		testAggregatedAPIServer(t, false, "1.1")
+		testAggregatedAPIServer(t, false, true, "1.1", "1.1")
 	})
-	t.Run("WithWardleFeatureGateAtV1.1", func(t *testing.T) {
-		testAggregatedAPIServer(t, true, "1.1")
+	// Testing emulation version N-1, BanFlunder default=false in 1.0
+	t.Run("WithoutWardleFeatureGateAtV1.0", func(t *testing.T) {
+		testAggregatedAPIServer(t, false, false, "1.1", "1.0")
```

Comment:
to avoid conflicts between the kube-apiserver featuregtes and the wardleapiserver featuregates are we better off directly doing CRUD against the wardle server instead of fully aggregating it for this particular set of tests?  I think the testing suite woudl be more understandable then and it would ease future maintenance.

Reply:
> That's a good suggestion. Can we defer this to later since we do not have any time left in this code freeze?

Yes, it is a test/bug fix.